File: src/productos/Electrodomestico.java
package productos;

public class Electrodomestico extends Producto {
    private String marca;

    public Electrodomestico(String nombre, float precio, String fechaImportacion, int numeroSerie, int stock, String marca) {
        super(nombre, precio, fechaImportacion, numeroSerie, stock);
        this.marca = marca;
    }

    public String getMarca() {
        return marca;
    }
}

File: src/productos/Electrodomestico.py
from productos.Producto import Producto


class Electrodomestico(Producto):
    def __init__(self, nombre, precio, fecha_importacion, numero_serie, stock, marca):
        super().__init__(nombre, precio, fecha_importacion, numero_serie, stock)
        self.__marca = marca

    def get_marca(self):
        return self.__marca
